src/index.html

```diff
@@ -4,9 +4,378 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__home-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="navigation">
+        <menu class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link"
+              href="#Laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#gadjets">Gadgets</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#video">Video</a>
+          </li>
+        </menu>
+      </nav>
+    </header>
+
+    <main class="content">
+      <section class="catalog content__catalog">
+        <article
+          class="card catalog__card"
+          data-qa="card"
+        >
+          <img
+            class="card__product-image"
+            src="./images/imac.jpeg"
+            alt="Imac picture"
```

Comment:
```suggestion
            alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
```

Reply:
maybe I misunderstood Andrey's comment, but he wrote to me to correct this name for a shorter one. 
![image](https://user-images.githubusercontent.com/123364948/217071052-0f1828d1-1aa5-42ed-b2e0-1b09bfa6e511.png)

Reply:
Read text carefully - it says `img is self-closing tag`. So instead of `<img></img>` you must use `<img />`.